llms/openai/openaillm.go

```diff
@@ -48,6 +52,16 @@ func (o *LLM) Generate(prompts []string, stopWords []string) ([]*llms.Generation
 	}, nil
 }
 
+// Chat requests a chat response for the given prompt.
+func (o *LLM) Chat(prompt string) (string, error) {
```

Comment:
I'm fine with this but we should consider what interface to define that would satisfy this.